### ISSUE TITLE
add liftr_info()

### ISSUE THREAD

**Issue by nanxstats** [road2stat/liftr#6]
liftrInfo() for replacing sessionInfo().

containing:
- Dcoker image info
- Rabix info
- Rabix workflow graph

**Comment by davidroberson** [comment]
This would be a great feature especially the rabix graph.

**Comment by nanxstats** [comment]
@davidroberson - thanks for the feedback, this will be implemented. It seems Rabix itself could make this graph, we plan to try this way first. Or, we could also parse the workflow information in R and plot it.

**Comment by tengfei** [comment]
@davidroberson yes, we will first try to render native rabix graphics, alternatively like Nan said, use basic R graphics to visualize directed graph workflow is also possible.
